test/common/upstream/load_balancer_fuzz.h

```diff
@@ -0,0 +1,53 @@
+#include "common/upstream/load_balancer_impl.h"
+
+#include "test/common/upstream/load_balancer_fuzz.pb.validate.h"
+#include "test/common/upstream/load_balancer_fuzz_test_utils.h"
+
+namespace Envoy {
+namespace Upstream {
+
+class LoadBalancerFuzzBase : public LoadBalancerFuzzTestBase {
+public:
+  LoadBalancerFuzzBase();
+
+  // Untrusted upstreams don't have the ability to change the host set size, so keep it constant
+  // over the fuzz iteration.
+  void initializeFixedHostSets(uint32_t num_hosts_in_priority_set,
+                               uint32_t num_hosts_in_failover_set);
+
+  virtual void initialize(test::common::upstream::LoadBalancerTestCase input) PURE;
+  void initializeAndReplay(test::common::upstream::LoadBalancerTestCase input);
```

Comment:
Why the whole test case? If the base class is viewed as just holding "replay logic" of parsing the protobuf and delegating actions, should it only contain the action trace?

Reply:
Oh yeah my question below is answered here, I guess this also maps to health check fuzz whenever I try and speed it up.

Reply:
Changed base class to construct static host sets, update host sets, consruct random, and call the load balancer.